Tests/Marketplace/marketplace_services.py

```diff
@@ -1820,21 +1823,21 @@ def collect_content_items(self):
                         })
 
                     elif current_directory == PackFolders.PARSING_RULES.value:
-                        self.is_siem_pack(content_item, 'ParsingRule')
+                        self.add_pack_type_tags(content_item, 'ParsingRule')
                         folder_collected_items.append({
                             'id': content_item.get('id', ''),
                             'name': content_item.get('name', ''),
                         })
 
                     elif current_directory == PackFolders.MODELING_RULES.value:
-                        self.is_siem_pack(content_item, 'ModelingRule')
+                        self.add_pack_type_tags(content_item, 'ModelingRule')
                         folder_collected_items.append({
                             'id': content_item.get('id', ''),
                             'name': content_item.get('name', ''),
                         })
 
                     elif current_directory == PackFolders.CORRELATION_RULES.value:
-                        self.is_siem_pack(content_item, 'CorrelationRule')
+                        self.add_pack_type_tags(content_item, 'CorrelationRule')
```

Comment:
This will obviously fail..... as this yml won't have the `isFetchEvents` field. 
We don't need a function here just do:
```
self._is_siem = True
```

Reply:
It should not fail... as the yaml_type is not Integration